sourced/ml/mining/__main__.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import logging
+import sys
+
+from modelforge import slogging
+
+from .cmd import ArgumentDefaultsHelpFormatterNoNone
```

Comment:
```suggestion
from sourced.ml.mining.cmd import ArgumentDefaultsHelpFormatterNoNone
```

We should always use the full import paths.

Reply:
Ok, will remember that.